src/SourceBuild/Arcade/eng/common/templates/job/source-build-run-tarball-build.yml

```diff
@@ -41,31 +41,36 @@ jobs:
         _EnablePoison: false
         _ExcludeOmniSharpTests: true
         _RunOnline: true
+        _WithPreviousSDK: false
       CentOSStream8-Offline:
```

Comment:
I thought the WithPreviousSDK leg was going to be a new leg only run in internal ci?